.github/workflows/new_prs.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         with:
           project: Openverse PRs
           column: In progress
-          repo-token: ${{ secrets.ACCESS_TOKEN }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
`GITHUB_TOKEN` [might not have access](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) to projects outside the repo.